src/api/hub.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export default class HubGroupAPI {
+  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io';
+  private readonly baseUrl: string;
+
+  private readonly apiPath = '/ent/api/templates/bundlegroups';
+
+  constructor(baseUrl?: string) {
+    this.baseUrl = baseUrl || this.defaultBaseUrl;
+  }
+
+  async callApi(uri: string): Promise<any[]> {
+    const response = await axios(`${this.baseUrl}${uri}`);
+    const { data } = response;
+
+    return data;
+  }
+
+  getHubGroups(): Promise<any[]> {
+    return this.callApi(this.apiPath);
+  }
+
+  getHubGroupById(versionId: number): Promise<any[]> {
```

Comment:
1) getBundleGroupById
2) This should return Bundle[]

Reply:
is it `Bundle` or `BundleGroup` for the type name?

Reply:
the return should be `Bundle[]`

Reply:
done